presto-docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -8,6 +8,17 @@ HTTP/HTTPS, using the corresponding port number specified in the coordinator
 :ref:`config_properties`. It can be configured with :ref:`specific related
 properties <web-ui-properties>`.
 
+You can disable the Web UI with the ``web-ui.enabled`` property, if you do not
+want to expose the information from the Web UI to your users.
+
+.. contents::
```

Comment:
No need for this, since the page is short.

Reply:
this is mostly in anticipation, but also because the page is longer than one screen at this stage. The page needs a LOT more info, but I removed for now.